W-14391130-distrbuted-tracing-RTM (#707)
mulesoft/docs-runtime-manager: runtime-manager/modules/ROOT/pages/deployment-strategies.adoc

=== Monitoring

[%header,cols="15,40,40,25"]
|===
|Deployment Option | Implementation       | Functionality| More Information
|CloudHub 2.0
 | - You can create alerts in xref:alerts-on-runtime-manager.adoc[Runtime Manager] for applications deployed to CloudHub 2.0. +

>    - You can enable distributed tracing for your CloudHub 2.0 applications that are deployed to either shared or private spaces and running on Mule runtime 4.5.1.e or later.

Reviewer: ```suggestion
   - You can enable trace data collection for your CloudHub 2.0 applications that are deployed to either shared or private spaces and running on Mule runtime 4.5.1.e or later.
```

Reviewer: This might need 4.6 - check?

  | - Run thread dumps. Configure application alerts. +
  +

>     - Once you enable distributed tracing, the Runtime Manager backend starts instrumenting traces for your application.

Reviewer: ```suggestion
    - Once you enable tracing, the Runtime Manager backend starts instrumenting traces for your application.
```

   | - xref:cloudhub-2::ch2-monitor-apps.adoc[Monitoring Apps Deployed to CloudHub 2.0] +

>      - xref:cloudhub-2::ch2-deploy-private-space.adoc#enable-distributed-tracing[Enable Distributed Tracing]

Reviewer: ```suggestion
     - xref:cloudhub-2::ch2-deploy-private-space.adoc#enable-distributed-tracing[Enable Trace Data Collection]
```
